src/pages/Home.jsx

```diff
@@ -38,6 +40,20 @@ const Home = () => {
       >
         <section id="home">
           <Stack alignItems="center" justifyContent="center">
+            <Box my={8}>
+              <Box
+                component="img"
+                src="/assets/PASSLogolightmode.png"
+                alt=""
+                sx={{ width: isReallySmallScreen ? '80%' : '300px', color: 'primary.main' }}
+              />
+              <Typography variant="h1" color="primary" fontWeight="600">
+                PASS
+              </Typography>
+              <Typography variant="h2" component="p" color="primary.main" fontWeight="500">
```

Comment:
I think this is better suited as `h3`. `h2` is too close to the `h1` of the PASS title, in my opinion.

Reply:
We need to keep it as an `h2` to maintain the heading hierarchy, so if we want to differentiate it _visually_, would need to modify the CSS for font size.

Reply:
@milofultz Do we?  I had it as a `<p>` as you can see on line 53.

Reply:
Gah, disregard that above comment, got totally mixed up on the props and things.

Hmm, actually am now wondering if this should be part of the `h1`, as then the `h1` would have context in terms of what PASS is ("PASS: Personal Access System for Services"). Don't know how nesting Typography elements works, but could do something like this:

```tsx
<Typography variant="h1" color="primary" fontWeight="600">
  PASS<span class="sr-only">: </span>
  <Typography variant="h2" component="span" color="primary.main" fontWeight="500">
    Personal Access System for Services
  </Typography>
</Typography>
```

Reply:
I will dig into this today sometime

Reply:
This link also has an idea for breadcrumbs under the red "Examples" section.

Reply:
> Gah, disregard that above comment, got totally mixed up on the props and things.
> 
> Hmm, actually am now wondering if this should be part of the `h1`, as then the `h1` would have context in terms of what PASS is ("PASS: Personal Access System for Services"). Don't know how nesting Typography elements works, but could do something like this:
> 
> ```tsx
> <Typography variant="h1" color="primary" fontWeight="600">
>   PASS<span class="sr-only">: </span>
>   <Typography variant="h2" component="span" color="primary.main" fontWeight="500">
>     Personal Access System for Services
>   </Typography>
> </Typography>
> ```
> 
> with `sr-only` being some variant of https://webaim.org/techniques/css/invisiblecontent/

@milofultz  if you feel like taking an attempt at a solution here, I've had zero luck in getting it to render the way it is now and be part of the `h1`. I think that is the correct idea, I just can't seem to solve.  You can just make a commit/push to this branch if you'd like.

I believe everything but this is currently sufficient in the branch (I still need to tackle the tests - which I'm digging into because I don't think they are the best tests for this component and may need to rewrite.)

Reply:
I'll make a PR so you can check it out and merge it into this one